modules/arriba/main.nf

```diff
@@ -2,15 +2,20 @@ process ARRIBA {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::arriba=2.1.0" : null)
+    conda (params.enable_conda ? "bioconda::arriba=2.2.1" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/arriba:2.1.0--h3198e80_1' :
-        'quay.io/biocontainers/arriba:2.1.0--h3198e80_1' }"
+        'https://depot.galaxyproject.org/singularity/arriba:2.2.1--hecb563c_2' :
+        'quay.io/biocontainers/arriba:2.2.1--hecb563c_2' }"
 
     input:
     tuple val(meta), path(bam)
     path fasta
     path gtf
+    path blacklist
+    path known_fusions
+    path structural_variants
+    path tags
+    path protein_domains
 
     output:
```

Comment:
You can add an output channel for the discarded fusion also

Reply:
Do you mean the fusion_fail one?